src/xo-app/dashboard/overview/index.js

```diff
@@ -288,35 +288,35 @@ export default class Overview extends Component {
             <Col mediumSize={4}>
               <Card>
                 <CardHeader>
-                  <Icon icon='pool' /> {_('poolPanel', { pools: this.props.nPools })}
+                  <Icon icon='pool' /> {_('poolPanel', { pools: props.nPools })}
                 </CardHeader>
                 <CardBlock>
                   <p className={styles.bigCardContent}>
-                    <Link to='/home?t=pool'>{this.props.nPools}</Link>
+                    <Link to='/home?t=pool'>{props.nPools}</Link>
                   </p>
                 </CardBlock>
               </Card>
             </Col>
             <Col mediumSize={4}>
               <Card>
                 <CardHeader>
-                  <Icon icon='host' /> {_('hostPanel', { hosts: this.props.nHosts })}
+                  <Icon icon='host' /> {_('hostPanel', { hosts: props.nHosts })}
                 </CardHeader>
                 <CardBlock>
                   <p className={styles.bigCardContent}>
-                    <Link to='/home?t=host'>{this.props.nHosts}</Link>
+                    <Link to='/home?t=host'>{props.nHosts}</Link>
                   </p>
                 </CardBlock>
               </Card>
             </Col>
             <Col mediumSize={4}>
               <Card>
                 <CardHeader>
-                  <Icon icon='vm' /> {_('vmPanel', { vms: this.props.nVms })}
+                  <Icon icon='vm' /> {_('vmPanel', { vms: props.nVms })}
                 </CardHeader>
                 <CardBlock>
                   <p className={styles.bigCardContent}>
-                    <Link to='/home?t=VM'>{this.props.nVms}</Link>
+                    <Link to='/home?s=&t=VM'>{props.nVms}</Link>
```

Comment:
Because the home view should display all vms by this link. Not only running vms.